### PR TITLE
[Merged by Bors] - feat: induction principle for Multiplicative

### PR DESCRIPTION
**Pull request by eric-wieser**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Command-Master — APPROVED** [review]

File: Mathlib/Algebra/Group/TypeTags.lean [hunk 1]
```diff
@@ -76,6 +76,11 @@ protected lemma «forall» {p : Additive α → Prop} : (∀ a, p a) ↔ ∀ a,
 @[simp]
 protected lemma «exists» {p : Additive α → Prop} : (∃ a, p a) ↔ ∃ a, p (ofMul a) := Iff.rfl
 
+/-- Recursion principle for `Additive`, supported by `cases` and `induction`. -/
+@[elab_as_elim, cases_eliminator, induction_eliminator]
```
Comment (b-mehta) [inline Mathlib/Algebra/Group/TypeTags.lean:80]: Is there a need to add @[eliminator] here?

**Review by b-mehta — APPROVED** [review]
bors d+

File: Mathlib/Algebra/Group/TypeTags.lean (hunk 1 above)
Comment (eric-wieser) [reply Mathlib/Algebra/Group/TypeTags.lean:80]: No, because `unknown attribute [eliminator]`; you must have hallucinated it existing.

**Comment by eric-wieser** [comment]
bors merge

**Pull request closed without merging**